scipy/stats/tests/test_distributions.py

```diff
@@ -5362,127 +5368,15 @@ def _rvs(self):
-        (stats.gennorm, (-1, )),
-        (stats.genpareto, (np.inf, )),
-        (stats.genexpon, (1, 2, -3)),
-        (stats.genextreme, (np.inf, )),
-        (stats.gausshyper, (1, 2, 3, -4)),
-        (stats.gamma, (-1, )),
-        (stats.gengamma, (-1, 0)),
-        (stats.genhalflogistic, (-1, )),
-        (stats.geninvgauss, (1, 0)),
-        (stats.gilbrat, ()),
-        (stats.gompertz, (-1, )),
-        (stats.gumbel_r, ()),
-        (stats.gumbel_l, ()),
-        (stats.halfcauchy, ()),
-        (stats.halflogistic, ()),
-        (stats.halfnorm, ()),
-        (stats.halfgennorm, (-1, )),
-        (stats.hypsecant, ()),
-        (stats.invgamma, (-1, )),
-        (stats.invgauss, (-1, )),
-        (stats.invweibull, (-1, )),
-        (stats.johnsonsb, (1, -2)),
-        (stats.johnsonsu, (1, -2)),
-        (stats.kappa4, (np.nan, 0)),
-        (stats.kappa3, (-1, )),
-        (stats.ksone, (-1, )),
-        (stats.kstwo, (-1, )),
-        (stats.kstwobign, ()),
-        (stats.laplace, ()),
-        (stats.laplace_asymmetric, (-1, )),
-        (stats.levy, ()),
-        (stats.levy_l, ()),
-        (stats.levy_stable, (-1, 1)),
-        (stats.logistic, ()),
-        (stats.loggamma, (-1, )),
-        (stats.loglaplace, (-1, )),
-        (stats.lognorm, (-1, )),
-        (stats.loguniform, (10, 5)),
-        (stats.lomax, (-1, )),
-        (stats.maxwell, ()),
-        (stats.mielke, (1, -2)),
-        (stats.moyal, ()),
-        (stats.nakagami, (-1, )),
-        (stats.ncx2, (-1, 2)),
-        (stats.ncf, (10, 20, -1)),
-        (stats.nct, (-1, 2)),
-        (stats.norm, ()),
-        (stats.norminvgauss, (5, -10)),
-        (stats.pareto, (-1, )),
-        # (stats.pearson3, (np.nan, )),
-        (stats.powerlaw, (-1, )),
-        (stats.powerlognorm, (1, -2)),
-        (stats.powernorm, (-1, )),
-        (stats.rdist, (-1, )),
-        (stats.rayleigh, ()),
-        (stats.rice, (-1, )),
-        (stats.recipinvgauss, (-1, )),
-        (stats.semicircular, ()),
-        (stats.skewnorm, (np.inf, )),
-        (stats.t, (-1, )),
-        (stats.trapezoid, (0, 2)),
-        (stats.triang, (2, )),
-        (stats.truncexpon, (-1, )),
-        (stats.truncnorm, (10, 5)),
-        # (stats.tukeylambda, (np.nan, )),
-        (stats.uniform, ()),
-        (stats.vonmises, (-1, )),
-        (stats.vonmises_line, (-1, )),
-        (stats.wald, ()),
-        (stats.weibull_min, (-1, )),
-        (stats.weibull_max, (-1, )),
-        (stats.wrapcauchy, (2, ))
-    ]
-)
-def test_support_gh13294_regression(dist, args):
+@pytest.mark.parametrize('distname, args', invdistdiscrete + invdistcont)
+def test_support_gh13294_regression(distname, args):
+    if distname in skip_test_support_gh13294_regression:
+        pytest.skip(f"skipping test for the support method for "
+                    f"distribution {distname}.")
+    try:
+        dist = getattr(stats, distname)
+    except TypeError:
+        dist = distname
```

Comment:
```suggestion
    dist = getattr(stats, distname)
```
Same as above, right?

Reply:
Ah, yes. Sorry, forgot about this.